spec/features/visitor_can_view_items_spec.rb

```diff
@@ -5,12 +5,14 @@
     let!(:item_one) {Item.create(name: "Canon 7d",
                                  description: "The best camera ever (NOT)",
                                  price: 15,
-                                 image_url: 'http://www.bhphotovideo.com/images/images1000x1000/sony_sel1635z_vario_tessar_t_e_16_35mm_1082052.jpg')}
+                                 image_url: 'http://www.bhphotovideo.com/images/images1000x1000/sony_sel1635z_vario_tessar_t_e_16_35mm_1082052.jpg',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.